### PR TITLE
disable browsing context name test

### PR DESCRIPTION
**Pull request by cbrewster**
---

- [ ] `./mach build -d` does not report any errors
- [ ] `./mach test-tidy` does not report any errors
- [ ] These changes fix #__ (github issue number if applicable).

- [ ] There are tests for these changes OR
- [ ] These changes do not require tests because _____

---

This change is [Reviewable](https://reviewable.io/reviews/servo/servo/12929)

### CODE REVIEW

File: tests/wpt/metadata/html/browsers/browsing-the-web/history-traversal/browsing_context_name.html.ini [hunk 1]
```diff
@@ -1,5 +1,3 @@
 [browsing_context_name.html]
   type: testharness
-  [Retaining window.name on history traversal]
-    expected: FAIL
-
+  disabled: see https://github.com/whatwg/html/issues/490
```
Comment (KiChjang) [inline tests/wpt/metadata/html/browsers/browsing-the-web/history-traversal/browsing_context_name.html.ini]: I'm kinda curious, how is this related?

Comment (metajack) [inline tests/wpt/metadata/html/browsers/browsing-the-web/history-traversal/browsing_context_name.html.ini]: The test looks for events for an iframe onload for each step. The initial frame has no src attribute, which means it will load about:blank. Based on whatwg/html#490, it looks like the behavior is unknown when the src changes before the onload event fires. That would make this test racy depending on the browser I'm guessing.

**Comment by KiChjang** [comment]
@bors-servo retry
* #12891

**Comment by ConnorGBrewster** [comment]
@bors-servo retry
 * #12891

**Comment by ConnorGBrewster** [comment]
@bors-servo retry
 * #8769
